src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Tabs } from '../components/Tabs';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId = '' } = useParams();
```

Comment:
You can use this parameter directly in the Tabs component to avoid roll it  as a prop

Reply:
Yeap, i agree with @DimaKolesnyk